test/nginx_system_test.sh

```diff
@@ -49,7 +49,11 @@ rm -rf "$TEST_TMP"
 mkdir -p "$TEST_TMP"
 echo TEST_TMP=$TEST_TMP
 
-APACHE_DOC_SRC="$MOD_PAGESPEED_DIR/src/install/"
+if [ -d "$MOD_PAGESPEED_DIR/src" ]; then
```

Comment:
Yes.  If someone has an existing ngx_pagespeed and mod_pagespeed checkout set up to work together (Otto) then they have mod_pagespeed checked out with a `src/` directory, and they're pointing `MOD_PAGESPEED_DIR` at `mod_pagespeed/` not `mod_pagespeed/src/`.  This change accepts checkouts that don't have `src/` without making the old checkouts stop working.